src/persistence/ContentPersistence.ts

```diff
@@ -11,6 +11,20 @@ import getRefUrl from "../content/getRefUrl";
 import convert from "../content/convert";
 import { Config } from ".";
 import { getDeepJoins } from "../content/rest/filterRefData";
+import { ContentFormat } from "../../typings";
+
+function findValueByPath(path: string | undefined, data: Cotype.Data) {
+  if (!path) return;
+
+  const titlePath = path.split(".");
+
+  const title = (titlePath.reduce(
+    (obj, key) => (obj && obj[key] !== "undefined" ? obj[key] : undefined),
```

Comment:
I know this was just moved. Is the `"undefined"` string intended? From just looking at this code, i'd assume it should be `undefined`.

Reply:
It seem as if a `typeof` is missing, otherwise the `"undefinded"` string would make no sense. Furthermore wouldn't just `obj ? obj[key] : undefined` do the trick?

Reply:
seems this way. But again maybe the API is not intuitive :)